src/Common/tests/System.Net/Sockets/Performance/SocketTestClient.cs

```diff
@@ -238,9 +238,23 @@ public Task<long> RunTest()
         }
 
         protected abstract string ImplementationName();
-        
-        //TODO: Either find a different fast way to compare or extract this in Interop.
```

Comment:
Yes, I expect this will meaningfully impact the test running time. 
We could think of ways to decouple correctness from performance measurements and remove this completely. Until then let's try to keep from diverging from the other variants (NetNative, Desktop).

Wouldn't a simple `#ifdef` block be enough to keep the old memcmp implementation for Windows only?